README.md

```diff
@@ -1,11 +1,78 @@
-# Bold BI
+# Bold BI Embedded Sample in .NET Core MVC
```

Comment:
This is common readme.md file for entire repo. You have to maintain your Readme.md file inside your folder alone. Revert this changes.

Reply:
Reverted the Readme.md file changes